cmd/tk/workflow.go

```diff
@@ -193,3 +193,40 @@ func stringsToRegexps(strs []string) (exps []*regexp.Regexp) {
 	}
 	return exps
 }
+
+func deleteCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete [path]",
+		Short: "delete the configuration to the cluster",
+		Args:  cobra.ExactArgs(1),
+		Annotations: map[string]string{
+			"args": "baseDir",
+		},
+	}
+	vars := workflowFlags(cmd.Flags())
+	force := cmd.Flags().Bool("force", false, "force deleteing (kubectl delete --force)")
```

Comment:
`force` and `autoApprove` are used in both, `apply` and `delete` so it makes sense to move their setup into a function (`func applyDeleteFlags(fs *pflag.FlagSet) `)

Reply:
It is better to split them into two functions for more feature in future, i do not think they are always same, WDYT?

Reply:
If one of them needs a specific flag only on itself, it can freely add it to its flagset. However, for me the fact that we do not need to maintain the same help message twice counts most so I would like to see these unified